cjworkbench/allauth_account_adapter.py

```diff
@@ -30,3 +34,21 @@ def send_confirmation_mail(self, request, emailconfirmation, signup):
         else:
             email_template = "account/email/email_confirmation"
         self.send_mail(email_template, emailconfirmation.email_address.email, ctx)
+
+    def add_message(
+        self, request, level, message_template, message_context=None, extra_tags=""
+    ):
+        """
+        Wrapper of `django.contrib.messages.add_message`, that reads
+        the message text from a template.
+        """
+        if "django.contrib.messages" in settings.INSTALLED_APPS:
+            try:
+                if message_context is None:
+                    message_context = {}
+                message_context.update(context_processor(request))
```

Comment:
Do not modify the input argument. Use `{**message, **context_processor(request)}` instead.